asteroidsyncserviced/bluez/bluezclient.h

```diff
@@ -20,15 +20,12 @@
 #ifndef BLUEZCLIENT_H
 #define BLUEZCLIENT_H
 
+#include "bluez_helper.h"
```

Comment:
I believe this is only required in bluezclient.cpp. If not, only include it in the header, not both the header and the compilation unit.

Reply:
It's needed in the header because it contains some typedefs that are needed for the interface.